platform/features/plot/src/PlotController.js

```diff
@@ -165,6 +184,19 @@ define(
                 }
             }
 
+            // Respond to a display bounds change (requery for data)
+            function changeDisplayBounds(event, bounds) {
+                self.pending = true;
+                releaseSubscription();
+                throttledRequery();
+                setBasePanZoom(bounds);
+            }
+
+            // Reestablish/reissue request for telemetry
+            throttledRequery = throttle(function () {
```

Comment:
Yes, throttling the event broadcasting is a much better approach to this. Waiting for the end of the user interaction still seems like overkill to me, but I think it will be fairly simple to wait for the bounds to become "stable" (e.g. user holds the bounds still for some number of ms) to get the desired effect. Think this will also remove the need for "request cancellation" since requests won't be issued until the bounds have stabilized.

If there are other cases where we specifically want to cancel out pending telemetry requests more generally, let's look at that in the context of #69.